### ISSUE TITLE
Can't set indentation when patching

### ISSUE THREAD

**Issue by letmaik** [marshallward/f90nml#79]
The indentation option can currently only be set when constructing a `Namelist` object. However, for the case when you want to patch a file, then there's no way to set the `indent` option to be used for the ouput patch file.

**Comment by marshallward** [comment]
I'm confused about this comment, because `f90nml.patch` is for replacing name-value strings inside an existing file, meaning that it should not actually change or modify the whitespace or any indentation.  Sometimes the substitution can look a bit odd, but that would not be address by the indentation of `Namelist`.

Can you show me what you're trying to do, and why you want to use `patch` and need to change the in

**Comment by letmaik** [comment]
With patch you can also add new keys, and for those it uses the default indentation.

**Comment by marshallward** [comment]
Right, thanks, that makes sense.  Hopefully not too difficult to implement.

**Comment by marshallward** [comment]
I haven't checked, but fairly sure it will create an empty list named `_delete`, which may or may not be ignored by the Fortran runtime.

In any case, I probably ought to do another pypi update soon, though it would be good to at least have a crack at the current outstanding issues around patching :).

**Comment by marshallward** [comment]
Actually, it seems it won't have any effect in older versions, since (as mentioned in #80) new content is simply ignored by `patch`.
